app/main.py

```diff
@@ -1,3 +1,38 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+import os.path
+
+from app.customer import Customer
+from app.shop import Shop
+
+
+def shop_trip() -> None:
+    current_directory = os.path.dirname(os.path.abspath(__file__))
+    file_path = os.path.join(current_directory, "config.json")
+    with open(file_path, "r") as file:
+        config = json.load(file)
```

Comment:
Do not overload the context manager.
The context manager is needed to work with the file, in our case, to read data. Therefore, after you have read the data from the file, exit the block.